config/targets-metadata.yml

```diff
@@ -0,0 +1,25 @@
+# Target files and metadata under top-level targets
+targets/fulcio_v1.crt.pem:
+  sigstore:
+    usage: Fulcio
+    status: Active
+targets/fulcio.crt.pem:
+  sigstore:
+    usage: Fulcio
+    status: Active
+targets/artifact.pub:
+  sigstore:
+    usage: Cosign
+    status: Active
+targets/ctfe.pub:
+  sigstore:
+    usage: CT
+    status: Active
+targets/rekor.pub:
+  sigstore:
+    usage: Rekor
+    status: Active
+targets/rekor.0.pub:
+  sigstore:
+    usage: Rekor
+    status: Active 
```

Comment:
Can we always include the new intermediate from https://github.com/sigstore/root-signing/issues/161#issuecomment-1099578582?

Reply:
Or in another PR?

Reply:
Adding it now! Might as well.